spec/clc/client_spec.rb

```
@@ -60,4 +60,36 @@
       end
     end
   end
+
+  describe '#list_datacenters' do
+    context "with cloud success", with_vcr('client/list_datacenters/success') do
+      it 'returns a list of datatacenters' do
+        expect(subject.list_datacenters).to be_an(Array)
+      end
+    end
+
+    context 'with cloud failure', with_vcr('client/list_datacenters/failure') do
+      it 'returns a list of datatacenters' do
+        expect(subject.list_datacenters).to be_an(Array)
```

Maybe it should throw an error here? How can this method work if cloud has failed?